code/modules/clothing/spacesuits/rig_modules/rig_modules.dm

```diff
@@ -166,11 +170,12 @@
 	..()
 
 /obj/item/rig_module/plasma_proof/can_install(var/obj/item/clothing/suit/space/rig/target)
-   if(!..())
-      return FALSE
+   var/parent_check=..()
+   if(!parent_check[1])
+      return list(FALSE,parent_check[2]) 
    if(target.clothing_flags & PLASMAGUARD)
-      return FALSE
-   return TRUE
+      return list(FALSE,"could not install the " + name +": suit is already plasma sealed.")
```

Comment:
please capitalize these outputs for consistency

Reply:
fixed, the capitals are now capitalized